cubano-httpeasy/src/main/java/org/concordion/cubano/driver/http/HttpEasy.java

```diff
@@ -234,7 +247,18 @@ public HttpEasy baseURI(String uri) {
     }
 
     /**
-     * Set the path part of the URL for the end-point.  baseURI, path and query are helpers only and any of these can take full URL.
+     * Instruct the current request to skip validation of any SSL certificates. Only applies to HTTPS connections.
```

Comment:
and just next request?

Reply:
Everything in HttpEasy if for the request and HttpEasyDefaults is global.